tests/unit/test_api.py

```diff
@@ -402,6 +402,11 @@ def test_get_ctms_for_api_example(client, example_contact):
         "/updates?start=2020-01-22T03%3A24%3A00%2B00%3A00&end=&limit=&after=&mofo_relevant=",
         None,
     ),
+    (
+        "GET",
+        "/updates?start=2020-01-22T03%3A24%3A00%2B00%3A00&end=&limit=&after=null&mofo_relevant=",
```

Comment:
As mentioned above, I think this should be a 422 instead of being successful. If we treat it as omitting ``after``, then it would restart the query sequence, and might look like our API is returning duplicate results.

Reply:
"null" is now invalid